src/App.jsx

```diff
@@ -1,12 +1,133 @@
-import './App.css'
+import "./App.css";
+import { Button } from "./components/Button/Button.jsx";
+import { Card } from "./components/Card/Card.jsx";
+import { useEffect, useState } from "react";
+import axios from "axios";
 
 function App() {
+  const [pokemons, setPokemons] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState(false);
+  const [pokemonDetails, setPokemonDetails] = useState([]);
```

Comment:
De naam`pokemonDetails` insinueert dat het over de details van één pokemon gaat, maar later in de code zie ik dat het de details van alle pokemon zijn, dus misschien is een andere naam beter?